crates/client/l1-messages/src/contract.rs

```diff
@@ -20,12 +20,17 @@ pub enum L1EventToTransactionError {
     InvalidEntryPointSelector(Felt252WrapperError),
     #[error("Failed to convert Nonce param from L1 Event: `{0}`")]
     InvalidNonce(Felt252WrapperError),
+    #[error("Failed to convert Messaging Address from L1 Event: `{0}`")]
+    InvalidMessagingAddress(Felt252WrapperError),
```

Comment:
Removed it because it was used while parsing the messaginAddress but I removed that field from L1HandlerMessage struct itself so not required right now.